x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/DeploymentManager.java

```diff
@@ -131,6 +133,14 @@ ProcessContext addProcessContext(Long id, ProcessContext processContext) {
     }
 
     public void startDeployment(TrainedModelDeploymentTask task, ActionListener<TrainedModelDeploymentTask> finalListener) {
+        startDeployment(task, null, finalListener);
```

Comment:
I overloaded this method to avoid making changes to existing tests which use argument captors